reframe/core/containers.py

```diff
@@ -119,32 +116,57 @@ class Sarus(ContainerPlatform):
     #: :default: :class:`False`
     with_mpi = fields.TypedField(bool)
 
+    #: Using metahub to fetch instance specific images
+    #:
+    #: :type: boolean
+    #: :default: :class:`False`
+    with_metahub = fields.TypedField(bool)
+    #: Skipping pull alltogether to speed up runs
+    #:
+    #: :type: boolean
+    #: :default: :class:`False`
+    skip_pull = fields.TypedField(bool)
+
     def __init__(self):
         super().__init__()
         self.with_mpi = False
         self._command = 'sarus'
+        self.with_metahub = False
```

Comment:
This is not needed since you can specify some hooks to be run before each step of the test pipeline.